### DISCUSSION TITLE
Web3Modal 4.1.1 for injected provider is broken

### DISCUSSION THREAD

**Discussion by nsnull0** [WalletConnect/walletconnect-monorepo#4357] [Developer Support]
Hello, i've been using 3.5.7 version just fine until i upgrade to latest 4.1.1
I have 2 problems:
1. `open wallet connection`
Only email login is working as expected!
all other provider such as injected provider is not working properly, when connected, it always need to refresh the page to make it work properly.
ref:
https://github.com/WalletConnect/web3modal/issues/2059

2. exposed config in logs
why it is exposing the project config? this never happen before in 3.5.7
Did i missed some steps? what is the correct way to do it? i am following the doc btw.
I think this data should not be exposed.
ref:
![Screenshot 2024-03-24 at 13 08 25](https://github.com/WalletConnect/walletconnect-monorepo/assets/26643922/8c3a850c-53d2-4293-bbe1-2c87ef080f50)

**Comment by glitch-txs** [comment]
Thanks! I answered 1 in the issue, for 2 I opened an issue we'll work on it, thanks for reporting!
https://github.com/WalletConnect/web3modal/issues/2071
